example/macos/AsyncStorageExample-macOS/AppDelegate+RNCAsyncStorageDelegate.m

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import "AppDelegate+RNCAsyncStorageDelegate.h"
+
+#import <RNCAsyncStorage/RNCAsyncStorage.h>
+
+@implementation AppDelegate (RNCAsyncStorageDelegate)
```

Comment:
Can we avoid duplicating `example/ios/AsyncStorageExample/AppDelegate+RNCAsyncStorageDelegate.m`?

Reply:
yes